### ISSUE TITLE
Cookbook feedback

### ISSUE THREAD

**Issue by Jeremiahstockdale** [permaweb/aos#320]
provide feeback on the cookbook, what could be more clear? are there obvious errors? etc.

**Comment by Jeremiahstockdale** [comment]
https://cookbook_ao.g8way.io/tutorials/bots-and-games/ao-effect.html
 ```
1.Install aos Fire up your terminal and run:

bash
npm i -g https://get_ao.g8way.io
```
seems to be outdated because of aos 2.0

**Comment by twilson63** [comment]
Yep, we have not released 2 yet once released we will update the get_ao under nameThxTomSent from my iPhoneOn Aug 8, 2024, at 12:54 PM, Jeremiah ***@***.***> wrote:﻿
https://cookbook_ao.g8way.io/tutorials/bots-and-games/ao-effect.html
1.Install aos Fire up your terminal and run:

bash
npm i -g https://get_ao.g8way.io

seems to be outdated because of aos 2.0

**Comment by dpshade22** [comment]
I agree that versioning the documents would be a significant improvement. 

Even a last updated, or "as of AOS 2.X" on each page would be a great improvement.

My devX has been hindered by not being confident in the documentation I'm using correct, especially given there are copies on Arweave that aren't up to date: https://ao_docs.ar.io/

**Comment by hitwill** [comment]
Another thing is that documentation on Spawning needs more information.
[Spawning a process](https://cookbook_ao.g8way.io/guides/aoconnect/spawning-processes.html#spawning-a-process)

It doesn't tell us what exactly a spawned process is. It tells you *how* to spawn a process but not *what* a spawned process is.

Is it a copy of the original process? 

Can a spawned process' handlers be updated?

What situations are useful for spawned processes?

**Comment by hitwill** [comment]
Links to some of autonomous finance's resources would also be very helpful:
https://github.com/Autonomous-Finance/aos-packages/tree/main/packages
https://github.com/Autonomous-Finance/ao-process-testing
https://github.com/Autonomous-Finance/aoform

**Comment by twilson63** [comment]
100%

@***@***.*** ***@***.***>
@***@***.*** ***@***.***>
@***@***.*** ***@***.***>

We should create a community guides section for

* Af
* Orbit
* BetterIdea
* etc

**Comment by twilson63** [comment]
All these suggestions are great! We are a small team!

Contributions to the cookbook are welcome!

https://github.com/permaweb/ao-cookbook

🙏

**Comment by hitwill** [comment]
@twilson63 raised an issue here https://github.com/permaweb/ao-cookbook/issues/231 (to start) and would like to push a pr, but the repo doesn't accept pushes or PR


`fatal: unable to access 'https://github.com/permaweb/ao-cookbook/'`
